controllers/auth.go

```diff
@@ -314,10 +313,11 @@ func (c *ApiController) Login() {
 
 			setHttpClient(idProvider, provider.Type)
 
-			if form.State != conf.GetConfigString("authState") && form.State != application.Name {
-				c.ResponseError(fmt.Sprintf("state expected: \"%s\", but got: \"%s\"", conf.GetConfigString("authState"), form.State))
```

Comment:
Is `authState` in app.conf not used anywhere after this change? Is yes, plz delete it everywhere

![image](https://user-images.githubusercontent.com/3787410/181794463-ec1092ff-db0c-4531-ba22-9a3ce27a7beb.png)